core/src/main/resources/struts-default.xml

```diff
@@ -376,6 +377,10 @@
                 <interceptor-ref name="alias"/>
                 <interceptor-ref name="servletConfig"/>
                 <interceptor-ref name="i18n"/>
+                <interceptor-ref name="cspInterceptor">
+                    <param name="enforcingMode">true</param>
+                    <param name="reportUri">csp-reports</param>
```

Comment:
We talked about not setting a reportUri by default so enabling csp by default (in report-only) mode doesn't generate extra traffic for the developers (just noting here so we don't forget )

Reply:
let's have this value start with a /, see comment on the settings object!

Reply:
resolved